spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'order'
+
+describe Order do
+
+  subject(:order) { described_class.new(menu_klass = menu, order_summary_klass = OrderSummary.new, sms_klass = sms)}
```

Comment:
Metrics/LineLength: Line is too long. [116/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)